app/Notifications/EmptyWikiNotification.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace App\Notifications;
+
+use App\Wiki;
+use Closure;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+use Illuminate\Support\Facades\Lang;
+use Illuminate\Support\HtmlString;
+
+/**
+ * Notification to be sent to empty Wikibase owner if the wiki stay empty longer than 30 days
+ */
+
+class EmptyWikiNotification extends Notification
+{
+    const TYPE = 'empty_wiki_notification';
```

Comment:
This is a very good place to put it :+1:

Reply:
I agree, although I keep wondering if some sort of reference in the `WikiNotificationSentRecord` Model would make sense, but I wouldn't know how.

Reply:
I can't think of any way myself right now either unfortunately.